app/flame_chart/flame_chart_model.ts

```diff
@@ -73,8 +101,62 @@ export function buildFlameChartModel(events: TraceEvent[], { visibilityThreshold
     currentThreadY += sectionHeight;
   }
 
+  let index = 0;
+  for (const { name, events } of timeSeries) {
+    const points = new Map<number, PointModel>();
+    const sectionHeight =
+      SECTION_LABEL_HEIGHT +
+      SECTION_PADDING_TOP +
+      SECTION_PADDING_BOTTOM +
+      TIME_SERIES_VERTICAL_GAP +
+      TIME_SERIES_HEIGHT;
+    let lowerBoundY = currentThreadY + sectionHeight;
+    let upperBoundY = currentThreadY + SECTION_LABEL_HEIGHT;
+    let d = `M 0 ${lowerBoundY} `;
+
+    const [color, lightColor] = getMaterialChartColorPairs(index);
+    const yMax = Math.max(...events.map((event) => event.value));
+    console.log(`yMax ${yMax}`);
+    console.log(`lowerBoundY ${lowerBoundY}`);
+    for (const event of events) {
+      const { name, ts, value } = event;
+      const x = ts / MICROSECONDS_PER_SECOND;
+      const y = lowerBoundY - (value / yMax) * TIME_SERIES_HEIGHT;
+      d += `L ${x} ${y} `;
+      points.set(Math.round(x), {
```

Comment:
Does this rounding mean that we are rendering the X values slightly inaccurately? (Do we need to round here?)